### PR TITLE
COMMERCE-2450 Order Placed cannot be found on Pending Orders and Placed Orders

### PR DESCRIPTION
**Pull request by riccardo-alberti**
ci:test

### CODE REVIEW

File: commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java [hunk 1]
```diff
@@ -458,8 +458,8 @@ public int getPlacedCommerceOrdersCount(
 
 		return commerceOrderLocalService.getCommerceOrders(
 			companyId, groupId, commerceAccountIds, keywords,
-			new int[] {CommerceOrderConstants.ORDER_STATUS_OPEN}, false, start,
```
Comment (marco-leo) [inline commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java:461]: @riccardo-alberti this is not a correct fix.  the method is getPlacedOrders . and was getting all orders that are not Open, why the change?

Comment (riccardo-alberti) [reply commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java:461]: @marco-leo i changed the method getUserPendingCommerceOrders so it uses the orderstastus that's already used in the admin section when listing the pending orders.

Comment (marco-leo) [reply commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java:461]: @riccardo-ferrari Admin UI and Front portlet have different logic and meanings, is correct that they show different data.

**Comment by marco-leo** [comment]
@riccardo-alberti this is not the proper fix
